Add support for the neighbors plugin

olipratt opened the pull request:
Hi,

This add support for the [neighbors](https://github.com/getpelican/pelican-plugins/tree/master/neighbors) plugin as per #97.

This:
- Only changes anything if you enable the 'neighbors' plugin.
- Uses the same code/style as the pagination buttons
- Minor fix of an overly long line and remove a duplicate `DEFAULT_PAGINATION` setting in `tests/pelicanconf.py`

**Note** - this adds new `Previous Post` and `Next Post` text, so don't know if this causes translation problems.

Tested on Windows 10 and Android 6, both using Chrome 56.0.2924.87 on articles with both buttons and first/last articles with only one button - all works as expected.

Here's a screenshot of the added buttons from the tests:
![prevnextpostbuttons](https://cloud.githubusercontent.com/assets/18227628/23578710/16f4d92a-00d5-11e7-91c2-37581d897483.png)

alexandrevicenzi commented:
@olipratt yes, it will need to be translated to all languages.

These buttons aren't placed well, they're so close. I think the should be placed at the corner of the page.

olipratt commented:
Hi @alexandrevicenzi 

Sorry for the delay I had a busy week.
I updated the CSS to remove the padding around the sides of the button container, and think it looks a lot better - hope that's what you meant. Checked it looks correct on PC and mobile.
I also updated the readme to note that the plugin is supported.

If you agree to take this change then I still need to regenerate the CSS file, but let me know what you think.

Thanks.

![buttonsatedge](https://cloud.githubusercontent.com/assets/18227628/24072909/2b11b0b0-0be6-11e7-8642-f1375f60774a.png)
